src/main/java/com/throughapinhole/photoportfolio/utils/Utils.java

```diff
@@ -5,7 +5,7 @@
  */
 final public class Utils {
     /** Value for a Category which has no parent */
-    final public static int CATEGORY_NO_PARENT = -1; 
+    final public static long CATEGORY_NO_PARENT = -1; 
```

Comment:
I'm guessing there will be more categories to be had later on. Better convert this to an enum instead.

Reply:
Ah, but this is just a constant value determining the categoryID of a category which has no parent, it will always be `-1`.